doc/page-features.h

```diff
@@ -104,6 +109,65 @@ Just to be clear: if you have several properties annotated with hasReadyFlag in
 you will not get indivdual signals for the ready flag of each of the properties, but get a
 \c readyFlagsChanged signal whenever the ready flag of a single property changes.
 
+\subsection sub5 Current behaviour
```

Comment:
Let's not document a behavior which we consider as broken.

Reply:
ok